components/engine_rocks/src/config.rs

```diff
@@ -366,8 +366,11 @@ impl FromStr for BlobRunMode {
             "normal" => Ok(BlobRunMode::Normal),
             "read-only" => Ok(BlobRunMode::ReadOnly),
             "fallback" => Ok(BlobRunMode::Fallback),
+            "kNormal" => Ok(BlobRunMode::Normal),
```

Comment:
> let's only support `normal/read-only/fallback`

In fact initial version only supports normal/read-only/fallback. However, I tested it did not work. Because there's some code path that requires the transition from  kNormal to BlobRunMode. It would complain kNormal is not supported even if in the cmdline, I used normal instead.  @Connor1996